packages/teleport-plugin-vue-base-component/src/utils.ts

```diff
@@ -117,7 +118,9 @@ export const generateRepeatNode = (
   const { dataSource, node: repeatContent, meta = {} } = node.content
   const repeatContentTag = generateNodeSyntax(repeatContent, accumulators)
 
-  let dataObjectIdentifier = meta.dataSourceIdentifier || `items`
+  const { key } = node.content.node.content as UIDLElement
```

Comment:
the child node is not necessarily an UIDLElement here, it can be any type of UIDLNode

Reply:
I tried having it as UIDLNode, but typescript is throwing error for that, saying --> Key does not exist on UIDLNode

Reply:
no sure if this is a solution, as we have no certainty that there's an element with a key there.

Reply:
All the nodes will have keys right? (I am not sure about this)

Reply:
only type element have keys, they are built based on the `name`, which again is only inside the content of element nodes.